backend/curriculum_tracking/tests/test_UserStatsPerWeekSerializer.py

```diff
@@ -215,13 +211,58 @@ def test_get_tilde_cards_reviewed_in_last_7_days(self):
         self.assertEqual(count, 1)
 
         # add another review, now the number of cards reviewed is 2
-
         review4 = RecruitProjectReviewFactory(
             timestamp=self.yesterday, reviewer_user=user
         )
         review4.timestamp = self.yesterday
         review4.save()
         AgileCardFactory(recruit_project=review4.recruit_project)
+        count = self.stats_serializer.get_tilde_cards_reviewed_in_last_7_days(user)
```

Comment:
You added count = self.stats_serializer.get_tilde_cards_reviewed_in_last_7_days(user), above it (line 210) the same assertion takes place and then later on again.  I think that either there should be one assertion or this test should be divided into three separate tests.  But seeing that they all test the same thing just at different stages I think the assertions are duplicated.

Reply:
Forgot to mention one thing that helps a lot:

When you make changes, in the description of the PR, it is a good idea to list
   the file names where you made changes and also the lines in which you made 
   changes, I saw that on one of Sheena's PR's and it makes reviewing a lot easier
   and quicker.  For instance, in your description you say you made changes to
   get_tilde_reviews_done_last_7_days and get_total_tilde_reviews_done but actually
   there are a lot of other small changes in other functions as well.

Reply:
I'll make the description more specific, but I'm not too sure about the specifying lines part. Isn't that what the diff on GitHub is for?  
Also, every file that has any changes is automatically shown in **Files changed**, why duplicate the process manually by mentioning files you made changes in?